### PR TITLE
Tox21 predict example with pickle

### PR DESCRIPTION
**Pull request by corochann**

### CODE REVIEW

**Comment by corochann** [comment]
Please merge after #139
I confirmed that I can save/load pickle model file and predict in python 3.5 env.
Note that since this PR breaks the example code not to work on library on PyPI (version 0.2.0). New version should be released ASAP after this PR merged.

File: examples/tox21/predict_tox21_with_classifier.py [hunk 1]
```diff
@@ -81,27 +77,14 @@ def main():
...
-        snapshot_file = _find_latest_snapshot(args.in_dir)
-    print('Loading pretrained model parameters from {}'.format(snapshot_file))
-    chainer.serializers.load_npz(snapshot_file,
-                                 predictor_, 'updater/model:main/predictor/')
-
-    clf = Classifier(predictor=predictor_, device=args.gpu,
-                     lossfun=F.sigmoid_cross_entropy,
-                     metrics_fun=F.binary_accuracy)
+    clf = Classifier.load_pickle(
+        os.path.join(args.in_dir, 'clf.pkl'),
```
Comment (mottodora) [inline examples/tox21/predict_tox21_with_classifier.py]: please be possible to specify file name by argparser.

File: examples/tox21/train_tox21.py [hunk 2]
```diff
@@ -182,6 +178,8 @@ def main():
     with open(os.path.join(args.out, 'config.json'), 'w') as o:
         o.write(json.dumps(config))
 
+    classifier.save_pickle(os.path.join(args.out, 'clf.pkl'),
```
Comment (mottodora) [inline examples/tox21/train_tox21.py]: ditto

**Comment by corochann** [comment]
add model-filename option, apply flake8

File: examples/tox21/train_tox21.py [hunk 3]
```diff
@@ -19,20 +18,16 @@
...
 except ImportError:
-    print('[WARNING] If you want to use BalancedSerialIterator, please install'
-          'the library from master branch.\n          See '
-          'https://github.com/pfnet-research/chainer-chemistry#installation'
-          ' for detail.')
-try:
-    from chainer_chemistry.training.extensions import ROCAUCEvaluator  # NOQA
-except ImportError:
-    print('[WARNING] If you want to use ROCAUCEvaluator, please install'
-          'the library from master branch.\n          See '
```
Comment (delta2323) [inline examples/tox21/train_tox21.py]: Does this PR solve #121 as well?

**Comment by delta2323** [comment]
Please add `predict_tox21_with_classifier.py` to the example test scripts (`example_test_cpu.sh` and `example_test_gpu.sh`)

**Comment by corochann** [comment]
I'd like to add test shell script in another PR.

File: examples/tox21/train_tox21.py (hunk 3 above)
Comment (corochann) [reply examples/tox21/train_tox21.py]: Yes

**Comment by delta2323** [comment]
As #139 is merged, could you merge or rebase the latest master branch?

**Comment by corochann** [comment]
resolved conflict

**Pull request merged**